src/librustdoc/html/markdown.rs

```diff
@@ -382,7 +382,6 @@ impl<'a, I: Iterator<Item = Event<'a>>> Iterator for LinkReplacer<'a, I> {
             Some(Event::Code(text)) => {
                 trace!("saw code {}", text);
                 if let Some(link) = self.shortcut_link {
-                    trace!("original text was {}", link.original_text);
```

Comment:
Why removing this `trace!`?

Reply:
The "original text" here isn't necessarily correct and now won't actually be used for matching anymore so it seems useless/misleading to trace it.

Reply:
:+1: